procrastinate/aiopg_connector.py

```diff
@@ -54,12 +58,12 @@ def _wrap_json(self, arguments: Dict[str, Any]):
         }
 
     @classmethod
-    async def create_with_pool_async(
+    def create_with_pool(
```

Comment:
To be honest, this method is pretty useless now. I think merging this with the init method would make things easier to understand.